src/Aspire.Hosting.Kubernetes/KubernetesPublisherOptions.cs

```diff
@@ -10,4 +10,66 @@ namespace Aspire.Hosting.Kubernetes;
 /// </summary>
 public sealed class KubernetesPublisherOptions : PublishingOptions
 {
+    /// <summary>
+    /// Gets or sets the name of the Helm chart to be generated.
+    /// </summary>
+    public string HelmChartName { get; set; } = "aspire";
```

Comment:
Should this be the name of the application instead of aspire?

Reply:
yeah! I just defaulted it to aspire, I thought the cli would probably pass it through